Patches/Magical Menagerie/MagicalMenagerie_CE_Patch_Races.xml

```diff
@@ -0,0 +1,1057 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>Magical Menagerie</modName>
+			</li>
+
+			<!-- Shared bodyShape patches -->
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>
+					Defs/ThingDef[
+						defName="MM_Ahuizotl" or
+						defName="MM_Chimera" or
+						defName="MM_ErymanthianBoar" or
+						defName="MM_Griffin" or
+						defName="MM_Kitsune" or
+						defName="MM_LernaeanHydra" or
+						defName="MM_Manticore" or
+						defName="MM_Pegasus" or
+						defName="MM_Qilin" or
+						defName="MM_Unicorn"
+					]
+				</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>Quadruped</bodyShape>
+					</li>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>
+					Defs/ThingDef[defName="MM_WillOWisp"]
+				</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>QuadrupedLow</bodyShape>
+					</li>
+				</value>
+			</li>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>
+					Defs/ThingDef[defName="MM_WildMinotaur"]
+				</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>Humanoid</bodyShape>
+					</li>
+				</value>
+			</li>
+
+			<!-- ========== Ahuizotl ========== -->
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="MM_Ahuizotl"]/statBases</xpath>
+				<value>
+					<MeleeDodgeChance>0.10</MeleeDodgeChance>
+					<MeleeCritChance>0.26</MeleeCritChance>
+					<MeleeParryChance>0.13</MeleeParryChance>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="MM_Ahuizotl"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>grabbing hand</label>
+							<capacities>
+								<li>MM_Grapple</li>
+							</capacities>
+							<power>15</power>
+							<cooldownTime>2</cooldownTime>
+							<linkedBodyPartsGroup>Hands</linkedBodyPartsGroup>
+							<!-- armor penetration stats not applicable here-->
```

Comment:
This lack of armor penn stats doesn't cause any issues, does it?

Reply:
Yes, this attack works fine without armor pen stats - it temporarily slows down the target pawn as a whole, rather than inflicting injuries on specific body parts.

No error messages popped during testing.